verify/src/metadata_schema.json

```diff
@@ -123,6 +123,28 @@
         },
         "perpendicular_baseline": {
           "type": "number"
+        },
+        "version": {
+          "type": "string"
+        },
+        "weather_model": {
+          "type": "string",
+          "enum": [
+            "None",
```

Comment:
It's unclear to me what the JPL metadata file will look like for v3 files that did not use a weather model at all, this assumes `weather_model: ['"None"]`, but that's worth confirming.